python_ta/__init__.py

```diff
@@ -272,20 +273,13 @@ def reset_linter(config=None, file_linted=None):
         ),
     )
 
+    parent_dir_path = os.path.dirname(__file__)
+    python_ta_dir = os.listdir(parent_dir_path)
+    checkers_index = python_ta_dir.index("checkers")
     custom_checkers = [
-        "python_ta.checkers.forbidden_import_checker",
-        "python_ta.checkers.possibly_undefined_checker",
-        "python_ta.checkers.global_variables_checker",
-        "python_ta.checkers.forbidden_io_function_checker",
-        "python_ta.checkers.invalid_range_index_checker",
-        "python_ta.checkers.one_iteration_checker",
-        "python_ta.checkers.type_annotation_checker",
-        "python_ta.checkers.unnecessary_indexing_checker",
-        "python_ta.checkers.shadowing_in_comprehension_checker",
-        "python_ta.checkers.redundant_assignment_checker",
-        "python_ta.checkers.invalid_for_target_checker",
-        "python_ta.checkers.missing_space_in_doctest_checker",
-        "python_ta.checkers.pycodestyle_checker",
+        ("python_ta.checkers." + os.path.splitext(f)[0])
+        for f in os.listdir(parent_dir_path + "/" + python_ta_dir[checkers_index])
+        if f != "__init__.py" and os.path.splitext(f)[1] != ""
```

Comment:
Check for the extension being `"py"` rather than `!= ""` (because the former is more specific)